scopes/react/component-highlighter/component-highlighter/component-highlighter.module.scss

```diff
@@ -1,11 +1,22 @@
 .highlighter {
+  overflow: scroll;
```

Comment:
are we sure we want this? not sure how it will look. 
also, maybe replace with `auto`?

Reply:
right!